components/TextFields/src/MDCTextField.m

```diff
@@ -254,6 +254,34 @@ - (void)applyCursorColor {
   self.tintColor = self.cursorColor;
 }
 
+#pragma mark - MDCLeadingViewTextInput Implementation
+
+- (void)setLeadingView:(UIView *)leadingView {
```

Comment:
unit test?

Reply:
Don't have time for unit tests so I made #4437